### DISCUSSION TITLE
Please Tell us the year you were born!

### DISCUSSION THREAD

**Discussion by throbotaith** [NAIST-SE/AttractingContributors#444] [General]
https://github.com/throbotaith/AttractingContributors-Years

If you do not wish to expose your personal information, false information is acceptable.
Please pull and modify the README.md and please do a pull request!

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter

**Reply by throbotaith** [reply]
OK

**Comment by throbotaith** [comment]
I would do pull request if you give the URLs
